webapp/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    presets: ['@babel/preset-env', '@babel/preset-react']
+}
```

Comment:
There's a missing trailing newline. I'd suggest installing the appropriate editorconfig.org plugin for your editor to prevent this. Missing this means future readers (who have their editors appropriately configured) who open the file are likely to force an unexpected change to the file.